src/components/CaseStudies/CaseStudyCard.tsx

```diff
@@ -54,7 +54,7 @@ export const CASESTUDIES: CaseStudy[] = [
     },
     {
         name: 'Criteo',
-        logo: '/external-logos/criteo-logo.svg',
+        logo: '/external-logos/criteo-logo.png',
```

Comment:
The new logo looks sweet! I did not notice any issues rendering it on `pricing`, `case-studies`, or `case-studies/criteo-tackles-big-code`. 

I noticed that this is the only PNG file we are using for a customer logo, while the other logos are SVG files. If we don't have a preference, would you be open to using the SVG from the issue instead? 

One advantage of using the SVG file is that we don't need to worry about resolution on different devices and screens. An SVG is also much smaller than the PNG, which can improve speed (see below). This [short article](https://www.growfox.co.uk/blog/5-reasons-you-should-be-using-svgs-over-pngs) provides a few more reasons as well.

<img width="529" alt="Screen Shot 2022-06-30 at 9 41 26 AM" src="https://user-images.githubusercontent.com/60713139/176706172-9c55952f-bc7e-4148-b596-c0a5e4ff6341.png">

Reply:
Good point! I think it makes sense for all our customer logos to be the file type too. I updated this in [f1a4dd2](https://github.com/sourcegraph/about/pull/5526/commits/f1a4dd2a24c98999c61eb036d1b1f5f43e750d94)